_episodes/01-introduction.md

```diff
@@ -40,5 +42,37 @@ install.packages("ggplot2")
 ~~~
 {: .r}
 
-[setup]: {{ site.baseurl }}/setup/
+### Prerequisites
+
+This lesson is designed for novice programmers that already have some practice
+with basic data management using base R function and the `dplyr` package. The
+following lessons, [Data Management in R][data-management] and 
+[`dplyr` Basics][dplyr], provide a refresher for the prerequisites and an 
+opportunity to prepare the [data][data] for the lesson examples.
+
+### Audience
+
+Have you ever seen a figure that tried to convey way too much? Or a figure that just didn't seem to give you any useful information?
```

Comment:
Add "Have you ever worried that you've created such figures yourself, and wondered if there was a reliable way to do better?"